Jenkinsfile_CNP

```diff
@@ -11,8 +11,11 @@ def secrets = [
     secret('test-law-firm-a-username', 'TEST_LAW_FIRM_A_USERNAME'),
     secret('test-law-firm-a-password', 'TEST_LAW_FIRM_A_PASSWORD'),
     secret('idam-redirect-uri', 'IA_IDAM_REDIRECT_URI'),
-    secret('idam-secret', 'IA_IDAM_SECRET'),
-    secret('s2s-name', 'S2S_NAME_WORKFLOW_API')
+    secret('idam-secret', 'IA_IDAM_SECRET')
```

Comment:
this might be for another ticket but we shouldn't be using the IA secret we have our own.
See the task-management-api to see where they are stored in our keyvault:
https://github.com/hmcts/wa-task-management-api/blob/79e979b84b993dc7899545a1e81953215f479247/infrastructure/main.tf#L49

Reply:
replied on Slack https://work-allocation.slack.com/archives/C016ELENM0S/p1604415246129600